build.sc

```diff
@@ -93,7 +96,35 @@ object os extends Module {
   }
 }
 
-trait OsLibModule extends CrossScalaModule with PublishModule {
+trait AcyclicModule extends ScalaModule {
+  def acyclicDep: T[Agg[Dep]] = T {
+    if (!ZincWorkerUtil.isScala3(scalaVersion())) Agg(Deps.acyclic)
+    else Agg.empty[Dep]
+  }
+  def acyclicOptions: T[Seq[String]] = T {
+    if (!ZincWorkerUtil.isScala3(scalaVersion())) Seq("-P:acyclic:force")
+    else Seq.empty
+  }
+  override def compileIvyDeps = acyclicDep
+  override def scalacPluginIvyDeps = acyclicDep
+  override def scalacOptions = T {
+    super.scalacOptions() ++ acyclicOptions()
+  }
+}
+
+trait SafeDeps extends ScalaModule {
+  override def mapDependencies: Task[coursier.Dependency => coursier.Dependency] = T.task {
+    val sd = Deps.scalaLibrary(scala213Version)
```

Comment:
Why not simply add this dependency when `scalaVersion().startsWith("2.13.")`? That would ensure the Scala 2.13 version is `>= scala213Version`.

Reply:
This is not working as it is a special case. `ScalaModule.mapDependency` replaces all occurences of a `scala-library` by the assumed correct version matching the `scalaVersion`, which can be either `scala-library` or `dotty-library`  or `scala3-library` and friends. 

Effectively, I'm only fixing the transitive older scala-library that comes in via Scala `3.1.3`. In case I build for Scala `2.13.10`, this fix isn't effective and not needed.